storage_service/locations/api/resources.py

```diff
@@ -248,6 +248,7 @@ class Meta:
             "used",
             "uuid",
             "verified",
+            "staging_path",
```

Comment:
Could we relocate the `staging_path` field above `used` to keep the list sorted alphabetically? Same below with the `filtering` dictionary key.